src/ResourceManager/Storage/AzureRM.Storage.Netcore.psd1

```diff
@@ -54,11 +54,11 @@ PowerShellVersion = '5.1'
 RequiredModules = @(@{ModuleName = 'AzureRM.Profile.Netcore'; ModuleVersion = '0.10.0'; })
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = 'netcoreapp2.0\Microsoft.Azure.Management.Storage.dll',
-    'netcoreapp2.0\Microsoft.Data.Edm.dll',
-    'netcoreapp2.0\Microsoft.Data.OData.dll',
-    'netcoreapp2.0\Microsoft.WindowsAzure.Storage.dll',
-    'netcoreapp2.0\System.Spatial.dll'
+RequiredAssemblies = '.\Microsoft.Azure.Management.Storage.dll',
+    '.\Microsoft.Data.Edm.dll',
```

Comment:
FYI, ".\" paths were not working in Linux in one of the beta builds. Please make sure this will note break Linux modules.

Reply:
This was fixed, and it works perfectly fine in linux. :)